cli.py

```diff
@@ -256,7 +256,7 @@ def cmd_build(self):
 
         rpmbuild_root = mkdtemp()
         sh.rpmbuild('--define', '%%_topdir %s' % rpmbuild_root,
-                    '-rp', kernel_src_rpm)
+                    '-rp', '--nodeps', kernel_src_rpm)
```

Comment:
cli.py: do not verify build dependencies   ---> 
1. using cli prefix
2. cli: don't check dependency when extract kernel source rpm